macos/FluentUITestViewControllers/TestLabelViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+import FluentUI
+
+
```

Comment:
extra newline

Reply:
I see double newlines a lot in this PR. Super minor but I don't normally see that, but maybe it's a swift thing I don't know about?

Reply:
No, that's a Saad thing. I should just addthe SwiftLint build phase and be done with it...